playbooks/tests/tasks/lbaas.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: tests lbaas
+  hosts: controller[0]
+  vars:
+    image_name: cirros
+    flavor_name: m1.tiny
+    pub_net: external
+    pri_net: internal
+    sub_net: internal_v4
+    stack_name: test_lbaas
+  tasks:
+  - copy: src={{ lookup('env', 'URSULA_ENV') }}/../..//playbooks/tests/tasks/lb_stack.yml dest=/tmp/lb_stack.yml owner=ubuntu group=ubuntu
+  - name: make sure admin has no role - heat_stack_user
+    shell: . /root/stackrc; openstack role add --user admin --project admin heat_stack_user; openstack role remove --user admin --project admin heat_stack_user;
```

Comment:
@j2sol I verified that if one user has no this role, this command cannot  fail, either.
And currently after ursula deployed cloud, admin will have this role default. If I don't remove this role, test will be failed.

It should be a bug, right? If so, where should we let admin have not this role while deploying?